### PR TITLE
New package: spex-x replaces spek-alternative for ffmpeg6

### PR DESCRIPTION
**Pull request by zlice**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, **x86_64**

### CODE REVIEW

File: srcpkgs/spek-x/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ wxWidgets-common wxWidgets-gtk3-devel"
+makedepends="ffmpeg6-devel wxWidgets-gtk3-devel"
+depends="ffmpeg6"
+short_desc="Acoustic spectrum analyser"
+maintainer="zlice <zlice555@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/MikeWang000000/spek-x"
+distfiles="https://github.com/MikeWang000000/spek-x/archive/v${version}.tar.gz"
+checksum="7f4b109903333cdd310ab2b9176f0dc49a565891376e281a233d342cedf90ef7"
+replaces="spek-alternatives<=0.8.2.3_2"
```
Comment (classabbyamp) [inline srcpkgs/spek-x/template:16]: don't use replaces and a dummy package together, just use the dummy package
```suggestion
```

File: srcpkgs/spek-x/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+# Tell autogen.sh to use "wx-config-gtk3" instead of "wx-config"
+export WX_CONFIG_NAME="wx-config-gtk3"
+
+pre_configure() {
+	NOCONFIGURE=1 ./autogen.sh
+}
+
+spek-alternative_package() {
+	depends="spek-x"
```
Comment (classabbyamp) [inline srcpkgs/spek-x/template:28]:
```suggestion
	depends="${sourcepkg}"
```

File: srcpkgs/spek-x/template [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+CXXFLAGS="-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64"
+
+# Tell autogen.sh to use "wx-config-gtk3" instead of "wx-config"
+export WX_CONFIG_NAME="wx-config-gtk3"
+
+pre_configure() {
+	NOCONFIGURE=1 ./autogen.sh
+}
+
+spek-alternative_package() {
```
Comment (classabbyamp) [inline srcpkgs/spek-x/template:26]: missing subpackage symlink

Comment (zlice) [reply srcpkgs/spek-x/template:26]: spek1

spek2


not sure what you mean, different than this?

Comment (classabbyamp) [reply srcpkgs/spek-x/template:26]: ah missed it

**Pull request merged**